test/e2e/pod_create_test.go

```diff
@@ -559,4 +560,217 @@ ENTRYPOINT ["sleep","99999"]
 		podJSON := podInspect.InspectPodToJSON()
 		Expect(podJSON.CPUSetCPUs).To(Equal(in))
 	})
+
+	It("podman pod create with --userns=keep-id", func() {
+		podName := "testPod"
+		podCreate := podmanTest.Podman([]string{"pod", "create", "--userns", "keep-id", "--name", podName})
+		podCreate.WaitWithDefaultTimeout()
+		Expect(podCreate.ExitCode()).To(Equal(0))
```

Comment:
This pattern is anti-recommended, please see #10997. Could you fix it (and all instances) to use the recommended form (`Expect(podCreate).To(Exit(0))`)? Thank you!

Reply:
Fixed